superset-frontend/src/views/CRUD/data/dataset/AddDataset/Header/index.tsx

```diff
@@ -59,21 +59,23 @@ const renderOverlay = () => (
 
 export default function Header({
   setDataset,
-  datasetName,
+  title,
+  schema,
 }: {
   setDataset: React.Dispatch<DSReducerActionType>;
-  datasetName: string;
+  title: string;
+  schema?: string | null | undefined;
 }) {
   const editableTitleProps = {
-    title: datasetName,
-    placeholder: t('Add the name of the dataset'),
+    title: schema ? title : t('New dataset'),
```

Comment:
curious why the usage of title here is tied to if schema is defined?  If you make title optional, then set a default value for title as t('New dataset') could the conditional logic here could be removed?

`
export const DEFAULT_TITLE = t(New dataset);
export default function Header({
  setDataset,
  title = DEFAULT_TITLE,
}: {
  setDataset: React.Dispatch<DSReducerActionType>;
  title?: string;
}) {
  const editableTitleProps = {
    title,
    placeholder: DEFAULT_TITLE
`

It looks like the only use of schema prop is this case so changing this logic might also make the prop unnecessary and simplify the component's interface

Reply:
Updated in [`this commit`](https://github.com/apache/superset/pull/21557/commits/e9f6eb28b1a684b08eb097d43dee99034d775df8)